index.html

```diff
@@ -110,7 +98,122 @@ <h3 id="gold-sidebar">
     </aside>
     <main id="main" class="main">
       <section id="home">
-        <h1>Call For Participation</h1>
+        <section class="highlight">
+          <p>December 5-7, Brussels, Belgium at <a href="http://www.digityser.org/" target="_blank">DigiTyser</a></p>
+          <p>Discover <a href="https://www.w3.org/2002/09/wbs/1/webvr-authoring/results#xbio" target="_blank">who's coming</a> and <a href="https://www.w3.org/2002/09/wbs/1/webvr-authoring/" target="_blank">register</a> now!</p>
+        </section>
```

Comment:
the layout of the hightlight box doesn't work well in large viewports where it pushes all the content below the right hand-side side bar

Reply:
Large viewports? Can you attach a screenshot, please?

Reply:
![viewport-vr](https://user-images.githubusercontent.com/216410/32849082-76e66dbc-ca2e-11e7-9533-223a635666ca.png)